src/public/CowSwap.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "CoW Swap",
-  "timestamp": "2022-03-01T14:54:18+00:00",
+  "timestamp": "2022-12-015T19:00:00+00:00",
   "version": {
-    "major": 0,
+    "major": 1,
```

Comment:
Should we increment major if it's just adding a token? 

See https://github.com/Uniswap/token-lists#semantic-versioning

Reply:
This was rebased on the wrong version of main. The current version is [1.0.0](https://files.cow.fi/tokens/CowSwap.json) and so adding a token should just increment the patch to 1.0.1. Will rebase on top of main